diff --git a/diskcache/src/CMakeLists.txt b/diskcache/src/CMakeLists.txt
index c555aa8b7..256f45429 100644
--- a/diskcache/src/CMakeLists.txt
+++ b/diskcache/src/CMakeLists.txt
@@ -185,7 +185,6 @@ cx_target_library(
   CURRENT ${libdiskcache_la_CURRENT}
   REVISION ${libdiskcache_la_REVISION}
   AGE ${libdiskcache_la_AGE}
-  STATIC
   SOURCES ${libdiskcache_la_SOURCES}
   LIBRARIES ${libdiskcache_la_LIBADD}
   INCLUDE_DIRECTORIES BEFORE PRIVATE
diff --git a/diskcache/src/Cache/CMakeLists.txt b/diskcache/src/Cache/CMakeLists.txt
index 45a974741..d7484777f 100644
--- a/diskcache/src/Cache/CMakeLists.txt
+++ b/diskcache/src/Cache/CMakeLists.txt
@@ -83,7 +83,6 @@ cx_link_headers( ${HDR_DIR} ${cache_HEADERS} )
 cx_target_library(
   TARGET cache
   CONVENIENCE
-  STATIC
   SOURCES ${libcache_la_SOURCES}
   INCLUDE_DIRECTORIES BEFORE PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}
diff --git a/diskcache/src/Common/CMakeLists.txt b/diskcache/src/Common/CMakeLists.txt
index 173b6150b..8156d186b 100644
--- a/diskcache/src/Common/CMakeLists.txt
+++ b/diskcache/src/Common/CMakeLists.txt
@@ -57,7 +57,6 @@ cx_link_headers( ${HDR_DIR} ${common_HEADERS} )
 cx_target_library(
   TARGET common
   CONVENIENCE
-  STATIC
   SOURCES ${libcommon_la_SOURCES}
   INCLUDE_DIRECTORIES BEFORE PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}
diff --git a/diskcache/src/Streams/CMakeLists.txt b/diskcache/src/Streams/CMakeLists.txt
index 97e3cf7c2..a76b713de 100644
--- a/diskcache/src/Streams/CMakeLists.txt
+++ b/diskcache/src/Streams/CMakeLists.txt
@@ -59,7 +59,6 @@ cx_link_headers( ${HDR_DIR} ${streams_HEADERS} )
 cx_target_library(
   TARGET streams
   CONVENIENCE
-  STATIC
   SOURCES ${libstreams_la_SOURCES}
   INCLUDE_DIRECTORIES BEFORE PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}
